addons/spreadsheet/static/src/public_readonly_app/public_readonly.js

```diff
@@ -35,17 +39,36 @@ export class PublicReadonlySpreadsheet extends Component {
         onWillStart(this.createModel.bind(this));
     }
 
+    get showFilterButton() {
+        return (
+            this.props.mode === "dashboard" &&
+            this.globalFilters.length > 0 &&
+            !this.state.isFilterShown
+        );
+    }
+
+    get globalFilters() {
+        if (!this.data.globalFilters || this.data.globalFilters.length === 0) {
```

Comment:
can't you get the filters from the model instead making `data` a component property?
`this.model.getters.getGlobalFilters()`

Reply:
> can't you get the filters from the model instead making `data` a component property? `this.model.getters.getGlobalFilters()`

@LucasLefevre I cannot use this getter here. It doesn't in `model.getters`. Probably because it's not in o-spreadsheet itself.

Reply:
Ah yes, true. The plugin isn't loaded. That's fine then